unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/WebRequest/WebRequestController.cs

```diff
@@ -298,9 +296,11 @@ public static WebRequestController Create()
             bool isReadable = true,
             Dictionary<string, string> headers = null)
         {
+            textureFactory.isReadable = isReadable;
+            return SendWebRequest(textureFactory, url, null, OnSuccess, OnFail, requestAttemps, timeout, disposeOnCompleted, headers);
             // Note: textureFactory is not used because of the known issue introduced in Unity 2021.3.17f1 using the UnityWebRequestTexture.GetTexture method
             // textureFactory.isReadable = isReadable;
-            return SendWebRequest(getWebRequestFactory, url, null, OnSuccess, OnFail, requestAttemps, timeout, disposeOnCompleted, headers);
+            // return SendWebRequest(getWebRequestFactory, url, null, OnSuccess, OnFail, requestAttemps, timeout, disposeOnCompleted, headers);
```

Comment:
Just drop the lines instead of commenting them

Reply:
I missed that, sorry 👍